build.sbt

```diff
@@ -87,7 +87,7 @@ lazy val lorre = (project in file("conseil-lorre"))
     javaExtras = Seq("-Xmx512M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
   )
   .enableAssembly()
-  .dependsOn(common, commonTestKit % Test)
+  .dependsOn(common % "compile->compile;test->test", commonTestKit % Test)
```

Comment:
This `test -> test` is what we wanted to avoid by introducing `commonTestKit` and that's the reason of these duplications. It complicates things actually, and I believe I had a talk with @ivanopagano about that. Mostly because (I don't remember exactly) but compile may trigger testing? We don't want to have it.

Reply:
So where should I put the `BitcoinFixtures` to be available in common and Lorre? (`commonTestKit` doesn't depend on common so I can't use the classes I need)

Reply:
for the moment you might need to put them in `common`?

Reply:
Currently `BitcoinFixtures` are in the common but in `test/` directory. Did you mean moving this to `main/`?

Reply:
My mistake, didn't realise that. I wonder if having fixtures for one module into another should be something we can avoid.

Are those fixtures also used in the common tests? Can we move them in the lorre module tests? Maybe we should simply duplicate those. It's an inter-modular dependency which we might want to loosen up for the sake of being more flexible.

Reply:
I've removed the dependency on the tests by creating a needed block row in Lorre without fixtures.